be/compendium/db/migrations/0001_initial.py

```diff
@@ -1,37 +1,58 @@
-# Generated by Django 3.0.14 on 2022-08-08 11:02
+# Generated by Django 3.0.14 on 2022-08-15 07:27
```

Comment:
- please avoid updating existing migration. Create a new migration instead

Reply:
I received an error while migrating in adding a new migration sir,

![image](https://user-images.githubusercontent.com/108708447/184827592-a3c81339-92ea-4183-82c2-c2bca77ce649.png)

 but i did found a solution which i tried in my copy of the project, but it's something I don't want to do in the main project
 
 
![image](https://user-images.githubusercontent.com/108708447/184827927-2a9d67d4-b09a-4e45-a416-871c72fddb38.png)

Reply:
I did created a new migration file sir, the error is that I can't migrate it to my database

![image](https://user-images.githubusercontent.com/108708447/184829922-3bc85c17-5b6c-4732-81cb-a387e8bfb368.png)